neqo-http3/src/connection.rs

```diff
@@ -995,6 +995,20 @@ impl Http3Connection {
         Ok(())
     }
 
+    /// Set the stream SendOrder.
+    /// # Errors
+    /// Returns InvalidStreamId if the stream id doesn't exist
```

Comment:
Clippy will complain here.

```suggestion
    /// Set the stream `SendOrder`.
    /// # Errors
    /// `InvalidStreamId` if the stream id doesn't exist.
```